src/Servant/OAuth/TokenServer.hs

```diff
@@ -38,23 +41,39 @@ import Servant.Server
 -- which should throw an error (using 'throwInvalidGrant') in the event of an invalid grant.
 tokenEndpointNoRefresh ::
   forall m grant claims contentTypes.
-  (MonadIO m, MonadError ServerError m, ToJWT claims) =>
+  (MonadIO m, MonadRandom m, MonadError ServerError m, ToJWT claims) =>
   JWTSignSettings ->
   (grant -> m claims) ->
   ServerT (OAuthTokenEndpoint' contentTypes grant) m
 tokenEndpointNoRefresh signSettings doAuth = \case
   Left _ -> throwServerErrorJSON err400 $ OAuthFailure InvalidGrantRequest (Just "unable to parse token request") Nothing
   Right grant -> do
     claims <- doAuth grant
-    tok <- liftIO $ makeAccessToken signSettings claims
+    tok <- makeAccessToken signSettings claims
     return $ OAuthTokenSuccess tok (jwtDuration signSettings) Nothing
 
+instance AsError ServerError where
```

Comment:
This is an orphan instance:
```
src/Servant/OAuth/TokenServer.hs:55:1: warning: [-Worphans]
    Orphan instance: instance AsError ServerError
    To avoid this
        move the instance declaration to the module of the class or of the type, or
        wrap the type with a newtype and declare the instance on the new type.
   |
55 | instance AsError ServerError where
```

Was it moved here from `ResourceServer`? :thinking: 
(At least I cannot spot orphans there. Though, the warnings for them are turned off: https://github.com/wireapp/servant-oauth-server/pull/5/files#diff-d884da9f13b462a8ef7c0cdbdc81c5d993ff7100d767ba6a91b78582015e23cdR10)